packages/web/components/view-mode.tsx

```
@@ -0,0 +1,41 @@
+          </button>
+          <button
+            onClick={() => onExport('markdown')}
+            className='px-3 py-1 text-sm border rounded'
+          >
+            Export
+          </button>
+        </div>
+      </div>
+      <div className='prose max-w-none'>
+        {typeof note.content === 'string' ? (
+          <div dangerouslySetInnerHTML={{ __html: note.content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)